src/wallet/wallet.cpp

```diff
@@ -1349,24 +1394,34 @@ void CWallet::SyncTransaction(const CTransaction& tx, const CBlock* pblock)
     MarkAffectedTransactionsDirty(tx);
 }
 
-void CWallet::MarkAffectedTransactionsDirty(const CTransaction& tx)
-{
+void CWallet::MarkAffectedTransactionsDirty(const CTransaction& tx) {
     // If a transaction changes 'conflicted' state, that changes the balance
     // available of the outputs it spends. So force those to be
     // recomputed, also:
-    BOOST_FOREACH(const CTxIn& txin, tx.vin)
+    BOOST_FOREACH(
+    const CTxIn &txin, tx.vin)
```

Comment:
If you're going to introduce line changes in this kind of code, then also migrate it to `for (const CTxIn &txin : tx.vin)`. Or revert the change.

Reply:
Accident, will fix.